import-service/src/services/s3-management-service.ts

```diff
@@ -50,7 +55,43 @@ export class S3ManagementService implements S3ManagementServiceInterface {
     }
   }
 
-  async moveFiles(objectKeys: string[]): Promise<unknown[]> {
+  //TODO:!!! types
+  async sendProductsToQueue(products: Product[]): Promise<any> {
+    try {
+      //TODO:!!! types
+      let results: any[] = [];
+
+      products.forEach(async (product) => {
```

Comment:
The **forEach** method is not suitable for executing asynchronous code. You cannot use **wait** for that code.

Reply:
fixed